### ISSUE TITLE
Convert absolute volumes to more flvolumes

### ISSUE THREAD

**Issue by joandrsn** [aau-giraf/giraf-production-swarm#8]
The `docker-compose.yml` file currently mounts everything to folder `/swarm-nfs/` folder. While this may be okay for the production server, running the compose file on a local system requires either the folder to exist or manually editing the file.

In some cases, the volumes linked are also files. When starting this, docker creates a folder with the filename. This is not ideal.

**Comment by Lortemanden** [comment]
There is a template for making issues. Please use it since its hard to follow what you want to change.

**Comment by joandrsn** [comment]
@Lortemanden I didn't find any of the points in the issue template relevant to this issue.

The point I'm trying to make is that: I cannot run this on my machine without changing a lot of lines in the configuration file.

I imagined this image to be a quick way for getting an local development environment set up for developing the front end.
